src/main/java/com/openlattice/chronicle/services/upload/AppDataUploadService.java

```diff
@@ -284,14 +290,16 @@ private EntitiesAndEdges getMetadataEntitiesAndEdges(
                     exception
             );
         }
-        metadataEntityData.put( edmCacheManager.getPropertyTypeId( START_DATE_TIME_FQN ),
-                entity.getOrDefault( START_DATE_TIME_FQN, Set.of( firstDateTime ) ) );
-        metadataEntityData.put( edmCacheManager.getPropertyTypeId( END_DATE_TIME_FQN ), Set.of( lastDateTime ) );
-        uniqueDates.addAll( entity.getOrDefault( RECORDED_DATE_TIME_FQN, Set.of() ) );
```

Comment:
Why is this being removed?

Reply:
RECORDED_DATE_TIME_FQN value will be updated with Merge, so there is no need to combine before upserting